ee/tabby-ui/lib/tabby/auth.tsx

```diff
@@ -270,8 +253,7 @@ function useSignOut(): () => Promise<void> {
   }
 }
 
-interface User {
-  isAdmin: boolean
+interface IJwtPayload {
```

Comment:
```suggestion
interface JwtPayload {
```

Prefer stating type directly, without `I` prefix